score/mw/com/impl/tracing/configuration/tracing_filter_config_test.cpp

```diff
@@ -727,5 +756,119 @@ TEST_F(TracingFilterConfigGetNumberOfTracingSlotsDeathTest, ConfigurationContain
     EXPECT_DEATH(tracing_filter_config_.GetNumberOfTracingSlots(configuration_.value()), ".*");
 }
 
+using TracingFilterConfigGetNumberOfTracingSlotsFixture = ConfigurationFixture;
+TEST_F(TracingFilterConfigGetNumberOfTracingSlotsFixture, InsertingNoTracePointsWithTraceDoneCBReturnsZero)
+{
+    const auto trace_point_type_0 = ProxyEventTracePointType::SUBSCRIBE;
+    const auto trace_point_type_1 = ProxyEventTracePointType::UNSUBSCRIBE;
+    const auto trace_point_type_2 = ProxyEventTracePointType::GET_NEW_SAMPLES;
+    const auto trace_point_type_3 = ProxyFieldTracePointType::GET;
+    const auto trace_point_type_4 = ProxyFieldTracePointType::SET;
+
+    // Given a valid config with tracing required for both events and fields, and trace points that do not require
+    // trace done callbacks
+    PrepareConfigurationWithEventsAndFields({"CurrentPressureFrontLeft"}, {"CurrentTemperatureFrontLeft"});
```

Comment:
Same for the titles of events and fields. Best you put them in constexpr global variables with an explicit name.
That avoid typos in your test and makes it easier to maintain.